src/components/login/LoginPage.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 
 import Header from '../common/Header';
+import LoginSection from './LoginSection';
+import WelcomeSection from './WelcomeSection';
+import '../../../public/assets/styles/login.scss';
```

Comment:
this is not how we agreed to import our scss files, refer to [this](https://github.com/andela/valinor-ah-frontend/blob/4d70aebf029554c024a3e0e1fa0117df12433849/public/styles/index.scss#L5)